convert/jenkinsjson/convertTestFiles/allure/fileOpsCopy.yaml

```diff
@@ -0,0 +1,24 @@
+steps:
```

Comment:
Please change this file name

Reply:
@jamie-harness, changed the file name, could you please review it now?